src/Loading.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Loading = () => {
```

Comment:
move this component to the component folder so that it can be used across the project.